README.md

```diff
@@ -1 +1,157 @@
+
+```
+docker run --shm-size="1024m" --rm -it lightly/worker:latest sanity_check=True
+```
+
+To run the worker remotely, it’s recommended to write a `run.sh` script with default parameters. Saving the command in a bash file is needed to run the worker in an automated way. 
+
+```
+docker run --shm-size="1024m" --gpus all --rm -i \
+    -e LIGHTLY_TOKEN={MY_LIGHTLY_TOKEN} \
+    lightly/worker:latest \
+    worker.worker_id={MY_WORKER_ID}
+```
+
+#### Grant Access to instance from SSM
+
+In order to send commands remotely to your instance, you need to grant AWS Systems Manager access to your EC2 Instance. You can do that by following the [AWS official documentation page](https://docs.aws.amazon.com/systems-manager/latest/userguide/setup-instance-profile.html#instance-profile-add-permissions). 
+
+### 1.2 Setting up the S3 Bucket
+
+If you don’t have an S3 bucket already, follow [the AWS official instructions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to create one. You will need two buckets, an `Input` and a `Lightly` bucket in order to work with Lightly. Then you have to [configure your datasource](https://lightly-docs.readme.io/docs/aws-s3#delegated-access) with delegated access in a way that Lightly can access your data.
+
+### 1.3 Local Setup
+
+These steps needs to be done in your local machine, where you will launch your pipeline that remotely controls your instance.
+
+Before you start, install the requirements that can be found in the `requirements.txt` file:
+
+```
+pip install -r requirements.txt
+```
+
+If you have never done it before, install AWS CLI in your local machine. You can find the tutorial on installing the AWS CLI in your system [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). Test if the installation went out correctly with the following:
+
+```
+which aws
+aws --version
+```
+
+In order to make your local machine control your automated pipeline, you need your local AWS CLI to have the proper permissions. You may need to perform a [quick setup](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html). In particular, be sure to configure it with an [AWS IAM User](https://us-east-1.console.aws.amazon.com/iamv2/home#/users) having the following Amazon AWS managed policies:
+
+- **AmazonEC2FullAccess**: to turn on and off your EC2 machine.
+- **AmazonS3FullAccess**: to upload your data on the S3 bucket.
+- **AmazonSSMFullAccess**: to remotely run the script.
+
+This should be your resulting AWS CLI user policy list:
+
+![Policy List](images/policies.png)
+
+If you followed correctly the steps in [Grant Access to instance from SSM](#grant-access-to-instance-from-ssm), you should be now able to reach your instance by SSM. Check if it shows in the SSM list with the following command. Be aware that it can take some hours for an instance to be added to the SSM list.
```

Comment:
```suggestion
If you followed the steps in [Grant Access to instance from SSM](#grant-access-to-instance-from-ssm) correctly, you should be now able to reach your instance by SSM. Check if it shows in the SSM list with the following command. Be aware that it can take some hours for an instance to be added to the SSM list.
```